### PR TITLE
Release v0.33.1

### PR DESCRIPTION
**Pull request by kddejong**
*Issue #, if available:*

*Description of changes:*
- Release v0.33.1

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,14 @@
+### v0.33.1
+###### Features
+- Add Python 3.9 testing (pull #[1584](https://github.com/aws-cloudformation/cfn-python-lint/pull/1584))
```
Comment (PatMyron) [inline CHANGELOG.md:3]:
```suggestion
```

[might still be issues with Python 3.9 not caught by the tests](https://github.com/aws-cloudformation/cfn-python-lint/issues/1520) and not added here yet:

https://github.com/aws-cloudformation/cfn-python-lint/blob/4b65edb615ab9e960d117219ffc619209baa5084/setup.py#L80-L81

so maybe we should hold off on mentioning 3.9?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,14 @@
...
+###### CloudFormation Specifications
+- Update CloudFormation specs to 15.0.0 (pull #[1582](https://github.com/aws-cloudformation/cfn-python-lint/pull/1582))
+- Add additional allowed values for `AWS::SecretsManager::SecretTargetAttachment.TargetType` (pull #[1573](https://github.com/aws-cloudformation/cfn-python-lint/pull/1573))
+- Add property types for `AvailabilityZone` in the `AWS::DMS::ReplicationInstance` and `AWS::EC2::Subnet` resources (pull #[1585](https://github.com/aws-cloudformation/cfn-python-lint/pull/1585))
```
Comment (PatMyron) [inline CHANGELOG.md:7]:
```suggestion
```

not sure this made it into [`CloudSpecs`](https://github.com/aws-cloudformation/cfn-python-lint/tree/master/src/cfnlint/data/CloudSpecs) to actually be in effect yet

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,14 @@
...
+- Expand allowed values for `AWS::CodeBuild::Project.Environment.Type` (pull #[1589](https://github.com/aws-cloudformation/cfn-python-lint/pull/1589))
```
[the remark this answers is not in the document]
Comment (kddejong) [reply CHANGELOG.md:6]: You're right.

File: CHANGELOG.md (hunk 2 above)
Comment (kddejong) [reply CHANGELOG.md:5]: Updated specs so its included now.

**Pull request merged**